powerauth-push-server/src/main/java/io/getlime/push/controller/rest/PushCampaignController.java

```diff
@@ -149,6 +153,7 @@ public PushCampaignController(PushCampaignRepository pushCampaignRepository,
      */
     @RequestMapping(value = "{id}/detail", method = RequestMethod.GET)
     @ResponseBody
+    @ApiOperation(value = "Return details about campaign", notes = "Specified with id")
```

Comment:
Use "details of a campaign", also provide more specific message in "notes", or remove "notes".